src/app/(proper_react)/redesign/(authenticated)/user/welcome/remove/page.tsx

```diff
@@ -0,0 +1,102 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { getL10n } from "../../../../../../functions/server/l10n";
+import { getOnerepProfileId } from "../../../../../../../db/tables/subscribers";
+import { getServerSession } from "next-auth";
+import { authOptions } from "../../../../../../api/utils/auth";
+import { NextResponse } from "next/server";
+import { optoutProfile } from "../../../../../../../external/onerep";
+
+export async function generateMetadata() {
+  const l10n = getL10n();
+  return {
+    title: "Welcome - Results", // FIXME l10n.getString("breach-meta-title"),
+    twitter: {
+      card: "summary_large_image",
+      title: l10n.getString("brand-fx-monitor"),
+      description: l10n.getString("meta-desc-2"),
+      images: ["/images/og-image.webp"],
+    },
+    openGraph: {
+      title: l10n.getString("brand-fx-monitor"),
+      description: l10n.getString("meta-desc-2"),
+      siteName: l10n.getString("brand-fx-monitor"),
+      type: "website",
+      url: process.env.SERVER_URL,
+      images: ["/images/og-image.webp"],
+    },
+  };
+}
+
+export default async function UserWelcomeRemove() {
+  const session = await getServerSession(authOptions);
+  if (!session?.user?.subscriber?.id) {
+    throw new Error("No session");
```

Comment:
We're going to replace this with a proper component that's based more closely on Figma, I don't want to build out a way to show errors to the users since that's happening in parallel elsewhere :)